external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/PostgresIntegrationSuite.scala

```diff
@@ -18,7 +18,7 @@
 package org.apache.spark.sql.jdbc
 
 import java.sql.Connection
-import java.util.Properties
+import java.util.{Properties, UUID}
```

Comment:
Do we use `UUID` here?

Reply:
Thanks! Remove the unused import statement.